playwright/tests/apps.spec.ts

```diff
@@ -18,45 +18,39 @@ test.beforeEach(({ page }) => {
 
 const PRE_INSTALLATION_TIMEOUT = 20 * 1000;
 const INSTALLATION_PENDING_TIMEOUT = 50 * 1000;
-const APP_LISTED_TIMEOUT = 15 * 1000;
-const APP_INSTALLED_TIMEOUT = 50 * 1000;
+const APP_EXPECT_UI_TIMEOUT = 15 * 1000;
```

Comment:
Do we need such long timeouts?

Reply:
https://github.com/saleor/saleor-dashboard/pull/5142/files#diff-6ee8effe67635adc776c9b91c22b9f9604535c2bf8475d5888b6c9d0031da57bR51